rustls/src/error.rs

```diff
@@ -76,7 +76,7 @@ pub enum Error {
     General(String),
 
     /// We failed to figure out what time it currently is.
-    FailedToGetCurrentTime,
+    FailedToTimeProvider,
```

Comment:
Guessing this is some kind of mass-rename fallout? 🙂 I'd expect the error variants to stick to `FailedToGetCurrentTime`.

Reply:
Weird, I did that rename due to a rebase conflict. Maybe I'm confused but I'll revert it as the original PR used `FailedToGetCurrentTime` as well.